src/vconsole/vconsole-setup.c

```diff
@@ -53,6 +54,41 @@ static bool is_vconsole(int fd) {
         return ioctl(fd, TIOCLINUX, data) >= 0;
 }
 
+static int toggle_utf8(int fd, bool utf8) {
```

Comment:
Yeah, diff can be a pain. The split mode (kdiff and friends, or the github interface) is usually better in those cases.